src/main/java/com/box/sdk/BoxTask.java

```diff
@@ -180,6 +180,15 @@ public Date getDueAt() {
         }
 
         /**
+         * Sets the task's due date.
+         * @param dueAt the task's due date.
+         */
+        public void setDueAt(Date dueAt) {
+            this.dueAt = dueAt;
+            this.addPendingChange("due_at", dueAt.toString());
```

Comment:
Use `BoxDateFormat` to ensure that the date is formatted in a way that the API can parse.